File: api/python/examples/http/rpc.py
#encoding=utf8  
import httplib
import json
httpClient = httplib.HTTPConnection('localhost', 15555, timeout=10)

#扩展HTTP头部
headers = {
    'cmd': 'produce', 
    'mq': 'MyRpc',
    'ack': False,
}
#JSON格式的HTTP消息体, method+params_list
req = {
    'module': 'Interface',
    'method': 'plus',
    'params': [1, 2]
}

httpClient.request('GET', '/', body=json.dumps(req), headers=headers)
 
print httpClient.getresponse().read() 

httpClient.close()

File: api/nodejs/examples/http/rpc.js
var http = require('http');

var data = {
	'method': 'getString',
	'params': ['test']
}
var body = JSON.stringify(data);

var options = {
	hostname: 'localhost',
	port: 15555,
	path: '/',
	method: 'POST',
	headers: {
		'mq': 'MyRpc',
		'cmd': 'produce',
		'ack': false,
		
		'content-length': body.length
	}
};

var req = http.request(options, function(feedback){
	if (feedback.statusCode == 200) {  
        var body = "";  
        feedback.on('data', function (data) { 
        	console.log(new String(data)); 
        })  
    }   
});

req.write(body);
req.end();
